WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -553,6 +579,107 @@ import CoreData
         
         try coreDataStore.saveIfNeeded(moc: backgroundContext)
     }
+    
+    // TODO: Grey
+    private func populateViewCountSlide(report: CDYearInReviewReport, backgroundContext: NSManagedObjectContext, project: WMFProject?) throws {
+        guard let iosFeatureConfig = developerSettingsDataController.loadFeatureConfig()?.ios.first,
+              let _ = iosFeatureConfig.yir(yearID: targetConfigYearID) else {
+            throw WMFYearInReviewDataControllerError.missingRemoteConfig
+        }
+
+        guard let userID else { return }
+        
+        let encodedUserId = "#" + String(userID)
```

Comment:
We're not encoding the "#" anymore, at least not at this point (we may be deeper in the networking stack), so maybe rename to `prefixedUserId`, or something like that. Also, I think it would be better to move this work into the `fetchEditViews(` method below, which makes the network call. This way callers of `fetchEditViews` don't need to remember to prefix a `#` to the userID.